### PR TITLE
Support and Test Against Django 5

### PR DESCRIPTION
**Pull request by albertyw**
Depends on #669 
Depends on #685 
Depends on #687 
Depends on #692 
Depends on #699

### CODE REVIEW

File: .github/workflows/test.yml
Comment (50-Course) [inline .github/workflows/test.yml:1]: Looks good to me 👍🏼

**Review by 50-Course — APPROVED** [review]
LGTM. Approving MR, since it does not introduce major breaking change.

**Comment by albertyw** [comment]
Looking at logs, it seems the tests are failing due to `ValueError: Another profiling tool is already active` which was reported in #682

**Review by 50-Course — APPROVED** [review]
LGTM, initially worried about the move from release candidate to stable, just confirmed it from the docs.

Pardon me, still work with older versions over here :)

**Pull request merged**